projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -115,6 +115,7 @@ public static function load_features() {
 		require_once __DIR__ . '/features/wpcom-hotfixes/wpcom-hotfixes.php';
 		require_once __DIR__ . '/features/wpcom-logout/wpcom-logout.php';
 		require_once __DIR__ . '/features/wpcom-themes/wpcom-theme-fixes.php';
+		require_once __DIR__ . '/features/launch-button/index.php';
```

Comment:
Nit: please keep this in alphabetical order. 

Although, I'd just add this feature to the existing `features/wpcom-admin-bar` for consistency 🤔

Reply:
I reordered this. It seems wpcom-admin-bar is simple site specific, so it might not be the right fit.

Reply:
> It seems wpcom-admin-bar is simple site specific

It's not. It affects Simple and Atomic.

Anyway, not a really big deal though.